website/docs/docs/deploy/model-notifications.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Model notifications"
+description: "While a job is running, receive email notifications in real time about any issues with your models. "
+---
+
+# Model notifications <Lifecycle status="beta" />
+
+Set up dbt to notify the appropriate model owners through email about issues as soon as they occur, while the job is still running. Model owners can specify which statuses to receive notifications about: 
+
+- `Success` and `Fails` for models
+- `Warning`, `Success`, and `Fails` for tests
```

Comment:
@rviswanathan-dbt : is it correct to use the back tick (code font) in this case? 

those are the literals, correct? i pulled these from the UI for the admin setting